Fixed something about notification and lockscreen.

michaellee123 opened the pull request:
1. Fixed album art image is not clear enough and always shows the previous song's cover;
2. Change Setting -> Personalize -> LOCKSCREEN (Show album cover, Blur album cover, Fullscreen controls) only about lockscreen, without change notification style anymore;
3. Lockscreen adapt landscape mode;
4. Move lockscreen image to the top, keep look like the player's style;
5. Change some defStyleAttr default value from -1 to 0, to make Android Studio to preview;
6. Fitting more pages can display circular buttons.

michaellee123 committed:
Fixed album art image is not clear enough and always shows the previous song's cover in Android 13 (test device: Sony Xperia 1 IV)

michaellee123 committed:
1. Fixed album art image is not clear enough and always shows the previous song's cover;
2. Change Setting -> Personalize -> LOCKSCREEN (Show album cover, Blur album cover, Fullscreen controls) only about lockscreen, without change notification style anymore;
3. Lockscreen adapt landscape mode;
4. Move lockscreen image to the top, keep looks like player's style;
5. Change some defStyleAttr default value frome -1 to 0, to make Android Studio to preview;
6. Fitting more pages can display circular buttons.

michaellee123 commented:
fix #1433
fix #1311

prathameshmm02 reviewed (changes requested):
@michaellee123 Blur lockscreen art setting should not affect the App's custom Lockscreen, what do you think @Daksh777? 

Everything else seems good, but I am a bit skeptical about removing of "Lockscreen art" and "Blur art" checks from MusicService. I think you should keep blur art check, user feedback would be helpful for you to improve that.

michaellee123 commented:
> 

Maybe I misunderstood, I thought the lock screen blur art meant the app's custom lock screen activity. But that's not why I modified it. It will cause some devices to display a notification cover art that is unclear and maybe the cover of the previous song. So I removed the check of MusicService, kept the switch in the settings page, and used it to control the app's custom lock screen activity.

prathameshmm02 committed:
Revert some changes in LockScreenActivity and MusicService

prathameshmm02 commented:
Hi @michaellee123, I've reverted some changes as mentioned above. Actually, the issue of blurry Album art in Android 13 was because of the wrong default value of isAlbumArtOnLockscreen, it should have been true, and because of that album art was loading properly. I have made that change too. And I am merging this now, thanks for your contribution.

michaellee123 commented:
@prathameshmm02 Thank you for merging my code. I'm so happy. But I made the app updated, and I had the same trouble with me, when if I toggle close the isAlbumArtOnLockscreen switch, the album art in the notification will go blurry, I think deleting the isAlbumArtOnLockscreen check is the way. In another way, if I keep it open, everything's well, so, let it be. :)

I'm trying to write a replay gain processor these days, it's not easy, and I'm very busy, maybe finished that in May or June.

prathameshmm02 commented:
Yeah, I think we should hide that preference on Android 13.

> I'm trying to write a replay gain processor these days, it's not easy, and I'm very busy, maybe finished that in May or June.

Yeah, that's great, looking forward to that feature.
